spec/syntax.md

```diff
@@ -318,15 +318,17 @@ that matches all values for a given _selector_.
 An **_<dfn>expression</dfn>_** is a part of a _message_ that will be determined
 during the _message_'s formatting.
 
-An _expression_ MUST begin with U+007B LEFT CURLY BRACKET `{` 
+An _expression_ MUST begin with U+007B LEFT CURLY BRACKET `{`
 and end with U+007D RIGHT CURLY BRACKET `}`.
 An _expression_ MUST NOT be empty.
-An _expression_ can contain an _operand_, 
-an _annotation_, 
+An _expression_ can contain an _operand_,
```

Comment:
I think we should promote "can" to normative "MUST" here. It was previously fine as "can" because there was nothing else permitted besides these three options.

```suggestion
An _expression_ MUST contain an _operand_,
```

Reply:
This is beyond the scope of this PR, which for this line is only removing the trailing space.

Reply:
Sure, that makes sense. I had not earlier caught the concern about an expression consisting of only attributes being non-empty.